src/content/docs/release-notes/docs-release-notes/docs-6-21-2024.mdx

```diff
@@ -0,0 +1,67 @@
+---
+subject: "Docs"
+releaseDate: '2024-06-21'
+version: 'version: June 14-20, 2024'
+---
+
+### New docs
+
+* We consolidated all of the errors inbox mobile crash docs into a single [mobile group crashes tab](/docs/errors-inbox/mobile-tab).
+* New [Analyze model data](/docs/ai-monitoring/explore-ai-data/view-model-data) describes the different ways you can use AI monitoring to observe your model performance. 
+* New [Google VertexAI monitoring integration](/docs/infrastructure/google-cloud-platform-integrations/gcp-integrations-list/google-vertexai-monitoring-integration) describes how to activate the integration and start monitoring it.
+* New [NVIDIA Jetson integration](/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration) describes how to monitor your NVIDIA Jetson device.
+
+### Major changes
+
+* In our Python agent configuration, added [startup debug](/docs/apm/agents/python-agent/configuration/python-agent-configuration/#NEW_RELIC_STARTUP_DEBUG) info.
+* In [mobile monitoring configure settings](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#android-background-reporting), added enable/disable info for background and `ApplicationExitInfo` reporting for Android.
+* A massive overhaul of the alerts navigation makes it easier for you to find what you're looking for in those docs.
+
+### Minor changes
+
+* Updated [iOS](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps) and [Android](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps) app launch time docs in response to customer feedback to make them more similar to one another.
+* [Infrastructure logs](/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent) now support Ubuntu 24.04.
```

Comment:
```suggestion
* [Infrastructure logs](/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent) now support Ubuntu 24.04 (Noble Numbat).
```

Reply:
my weird niche knowledge about ubuntu and learning how to troubleshoot ubuntu on my VM taught me that they refer to all their updates by """codenames""" that users like to use

Reply:
we also refer to them in our ubuntu docs too!

Reply:
Good call!